### PR TITLE
When nodesOnPath is undefined actions div should be hidden

### LINKED ISSUE #25: _getButtonLabel is not called when nodesOnNodePath is undefined
_getButtonLabel is not called when nodesOnNodePath is undefined .

This becomes a problem since Neovici/cosmoz-dialog#10 because nodesOnNodePath is undefined until the dialog template is stamped. 
@nomego Should we default `nodesOnNodePath` to null or empty Array ?

### PR DESCRIPTION
**Pull request by megheaiulian**
When nodesOnPath is undefined set button to same height as line-height normal.
Fixes #25

### CODE REVIEW

File: cosmoz-treenode-button-view.html [hunk 1]
```diff
@@ -44,7 +47,7 @@
 		</style>
 		<div class="horizontal layout">
 			<paper-button class="flex" raised on-tap="openDialogTree">
-				<div class="pathToNode">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
+				<div class="pathToNode" has-nodes$="[[ nodesOnNodePath.length ]]">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
 			</paper-button>
 			<paper-icon-button icon="clear" on-tap="reset" hidden$="[[ !_enableReset(nodePath, noReset) ]]"></paper-icon-button>
```
Comment (plequang) [inline cosmoz-treenode-button-view.html]: This is also an issue with `cosmoz-dialog` actually. The clear icon is visible until `nodePath` is set.
One solution could be to use a `visible` (i.e. opposite to hidden) attribute.  I mean the icon could have display: none by default, but become visible if `_enableReset` returns true

File: cosmoz-treenode-button-view.html [hunk 2]
```diff
@@ -44,7 +47,7 @@
 		</style>
 		<div class="horizontal layout">
 			<paper-button class="flex" raised on-tap="openDialogTree">
-				<div class="pathToNode">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
+				<div class="pathToNode" has-nodes$="[[ nodesOnNodePath.length ]]">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
```
Comment (plequang) [inline cosmoz-treenode-button-view.html]: This works, but does not look  really "adaptive".
What about having a `<div>&nbsp</div>`, inside the `paper-button` hidden when nodePath is set ?

**Comment by plequang** [comment]
@megheaiulian: forget about my previous review comments.
Instead, the whole `<div>` with the button and the clear icon should be hidden when `nodePath` is empty. See https://github.com/Neovici/cosmoz-treenode-navigator/issues/25#issuecomment-342639313

**Comment by megheaiulian** [comment]
@plequang Ready for review.

**Pull request merged**